src/main/scala/com/gwi/api/TaskRouter.scala

```diff
@@ -0,0 +1,90 @@
+package com.gwi.api
+
+import akka.actor.ActorSystem
+import akka.event.Logging
+import akka.http.scaladsl.marshalling.sse.EventStreamMarshalling._
+import akka.http.scaladsl.model.ContentTypes._
+import akka.http.scaladsl.model.StatusCodes._
+import akka.http.scaladsl.model.sse.ServerSentEvent
+import akka.http.scaladsl.model.{HttpEntity, Uri}
+import akka.http.scaladsl.server.{Directives, Route}
+import com.gwi.service.TaskService
+import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport
+import io.circe.syntax.EncoderOps
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.DurationInt
+import scala.util.{Failure, Success, Try}
+
+class TaskRouter(taskService: TaskService)(implicit ec: ExecutionContext, system: ActorSystem)
```

Comment:
Would be nice to test `TaskRouter`

Reply:
I've added some sample test for create endpoint.